lib/generators/wprof_generator.rb

```diff
@@ -0,0 +1,41 @@
+class WprofGenerator < Rails::Generators::Base
+  desc 'This generator creates an initializer file for Wor-Prof!'
+  def wor_prof_init
+    create_file 'config/initializers/wprof.rb', init_wprof
+  end
+
+  private
+
+  def init_wprof
+    "# -----------------------------------------
```

Comment:
Yo en estos casos usaría un Squiggly heredoc:

https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc

Las string multilinea con `"` no me ayudan a entender que es una string.

Reply:
Nice... i will try to apply it later.

Reply:
Done!! Add Squiggly heredoc in place to double quote.